Source/Core/Core/IOS/USB/Emulated/Skylanders/Skylander.cpp

```diff
@@ -1,7 +1,7 @@
-// Copyright 2022 Dolphin Emulator Project
+// Copyright 2023 Dolphin Emulator Project
```

Comment:
Don't update the copyright on existing files, please, you just moved it.

Reply:
The existing files in question were added to the project this year so I'd say they should be 2023.

Reply:
The PR for it was opened in 2022 so AdmiralCurtiss is right